app/definitions/rest/v1/rooms.ts

```diff
@@ -45,3 +45,7 @@ export type RoomsEndpoints = {
 		POST: (params: { roomId: string; notifications: IRoomNotifications }) => {};
 	};
 };
+
+export type TRoomsMediaResponse = {
```

Comment:
Not following the pattern just because it's not meant to use useEndpointData hook. It's just the best spot to place this type.